services/meta/system.md

```diff
@@ -33,7 +31,35 @@ in   | cmd.node_reinterview        | str_map    | Value example `{"max_age":"1"}
 in   | cmd.channel.get             | null    | Requests the current Zigbee channel.
 out  | evt.channel.report          | int     | Reports the current Zigbee channel. The value is 0 if the network has not been established yet.
 
-#### Notes
+## Service props
+
+| Name                | Type   | Description                                                                                  |
+|---------------------|--------|----------------------------------------------------------------------------------------------|
+| `sup_configuration` | object | Optional. List of supported configuration params, see [`configuration_param`](#definitions). |
+
+## Definitions
+
+* `configuration_param` is an object of the following structure:
+
+| Name            | Type   | Example                | Description                                                     |
+|-----------------|--------|------------------------|-----------------------------------------------------------------|
+| `param_id`      | int    | `2`                    | Id of the parameter.                                            |
+| `name`          | string | `"Temperature alarms"` | Name of the parameter.                                          |
+| `format`        | string | `"radio buttons"`      | Format of the parameter. One of [`param_format`](#definitions). |
+| `size`          | int    | `2`                    | Size of the parameter. Can be one of: 1, 2 or 4.                |
+| `min`           | int    | `-3`                   | Minimum value of the parameter.                                 |
+| `max`           | int    | `50`                   | Maximum value of the parameter.                                 |
+| `default`       | int    | `10`                   | Default value of the parameter.                                 |
+| `read_only`     | bool   | `true`                 | Defines whether parameter is non-modifiable.                    |
+| `advanced`      | bool   | `false`                | Defines whether parameter is advanced.                          |
+| `alters_device` | bool   | `false`                | Defines whether parameter will alter device capabilities.       |
```

Comment:
I am against adding a service which agregates in a sink all possible configurations which are not added to specific services.

Reply:
Following this way of thinking we could have just a one service with addr / cmd type / parametr type / value. It woul dbe too much generic. We need to try to provide nice-to-use API wth division to services, some logic units etc..

Reply:
Its adapter's role to translate parameters, group them to match specific service and consitute a logic part.